docs/charter.md

```diff
@@ -62,6 +64,10 @@ The _CPANSec_ intends to be a forum for coordinating and assisting in resolving
 The _CPANSec_ concerns itself with a limited domain:
 
 - Distributions published on CPAN, not including dual-life core modules;
+- CPAN distributions that function as shims (wrappers) for non-CPAN projects or resources. Examples:
+  - CPAN Distributions that are pure wrappers around compiled libraries (e.g. XS modules wrapping `libxml2` or `sqlite3`);
+  - CPAN Distributions (outside the `Alien::` namespace) that embed or use third-party (non-CPAN) projects or APIs directly;
```

Comment:
as it is written, it feels like:

```
- A;
- ¬A;
```

so we might as well join them (unless I misread something)

Reply:
Hm. Yes, this section requires a little more thought.

* The "shims" or "wrappers" terms imply a compilation step, meaning we concern ourselves just with the bits published on CPAN, and not any shared object files they enable the use of in Perl code. (e.g. XS modules wrapping `libxml2` or `sqlite3`);
    * Or do we also concern ourselves with out-of-ecosystem dependencies like these?
* The `Alien::` modules are different in the sense that they can (and do) actually download and install their dependencies locally, instead of relying on dependencies that are supplied by a native packaging system.
    * Do we care about Alien modules that may install vulnerable dependencies, even if these are not installed in a system location like `/usr/local/lib`?
* The second example is about "non-Alien" modules that embed their dependencies. These may include web framworks that contain Javascript code made part and distributed together with the CPAN dist tarball.
    * An example of this may be the bootstrap.js files that come with Mojolicious. If a vulnerability is discovered in a release of bootstrap.js that is embedded with Mojolicious, do we concern ourselves with this?

I think these are distinct enough examples that they are worth listing separately.